### DISCUSSION TITLE
which cuda version is used  in test env?

### DISCUSSION THREAD

**Discussion by wlaiso** [PaddlePaddle/PaddleSpatial#229] [BaiduKDDCup2022] [answered]
hi, jingbo or anyone: 
  can you tell me the cuda version in online env?

**Comment by shaido987** [comment, marked as answer]
Hi,

The cuda version in the online environment should be 11.2, see the answer to my previous question regarding installing some new packages: https://github.com/PaddlePaddle/PaddleSpatial/discussions/78#discussioncomment-2817399.
